app/models/service_ansible_playbook.rb

```diff
@@ -163,4 +164,15 @@ def decrypt_options(opts)
       opts[:extra_vars].transform_values! { |val| val.kind_of?(String) ? MiqPassword.try_decrypt(val) : val }
     end
   end
+
+  def log_stdout(action)
+    log_option = options.fetch_path(:config_info, action.downcase.to_sym, :log_output)
```

Comment:
@bzwei do we need a default value of on_error so it would work with existing entities?

Reply:
default here is not to log. The true default is expected to be set by the template definition which comes from UI.